superset-frontend/src/explore/actions/hydrateExplore.ts

```diff
@@ -96,6 +96,8 @@ export const hydrateExplore =
     if (dashboardId) {
       initialFormData.dashboardId = dashboardId;
     }
+
+    // const initialDatasource = dataset;
```

Comment:
I assume this is redundant
```suggestion

    // const initialDatasource = dataset;
```

Reply:
🤦